### DISCUSSION TITLE
Welcome to RuneKit Discussions!

### DISCUSSION THREAD

**Discussion by whs** [whs/runekit#8] [General]
Hi! and welcome to RuneKit discussion.

This discussion area is for user to user support. I might not be monitoring this section myself but hopefully some people would.

**Comment by vegantom** [comment]
I don't know if you know about this issue, but at the moment it doesn't work on X11 with gnome.

My friend suspects this is related to mutter.

Is this known already? And is this something intended to be patched?

The irony is if I load runescape in wayland (with my nvidia hardware) alt tool kit seems to load, but runescape does not work properly.

Regardless really lovely work, thank you for what you've done to get alt tool kit on unix systems!

**Reply by whs** [reply]
What exactly is not working? I'm on 4a23bcfedd21db34da0ba3e915e55f92e31627d0 and it seems to be working totally fine for me. I'm not using GNOME though but I can't think of any incompatibility.

**Reply by vegantom** [reply]
The app will find the rs session then get stuck, it didn't have this issue on plasma, seems to be gnome40 only according to my friend, the window / overlay for alt tool kit just isn't starting up. This is why a mutter issue is suspected. I'm not really sure what other info I can give but if there is a log or anything to look at please let me know :)

**Reply by whs** [reply]
The full log should be outputted if you start the app from terminal. For AppImage just use ./RuneKit

**Reply by vegantom** [reply]
```[tom@Arch-Tom Documents]$ sudo ./RuneKit.AppImage 
[sudo] password for tom: 
INFO:root:Starting QtWebEngine
QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-root'
INFO:runekit.game.x11.manager.X11GameManager:Found game instance 27262987```

This is obviously in x11 not wayland (as I say, xwayland doesn't want to work with the runescape client, but the tool kit appears to work there), gnome40, and I redownloaded the app image yesterday. It did behave I believe correctly on the initial attempt (downloading lots of toolkit things), but never opened a session fully. Not sure if it's possible to extract the version from the appimage but it should be the most up to date one from around less than a week ago.

Issue persists on a custom kernel and the mainline kernel.

**Reply by vegantom** [reply]
Jumped on to plasma-meta for now, works on x11, if you want me to go back to gnome for testing let me know.

**Comment by john-ghatas** [comment]
Thanks for the work on the port, running GNOME atm and I would love to get it working. Is there any way I can help by providing logs etc? If so please let me know what I can do.

**Reply by whs** [reply]
Is that GNOME on Wayland? It would be as much work as to support another operating system to support Wayland. I wrote [a note](https://old.reddit.com/r/runescape/comments/p3ag53/issues_with_runekit_on_linux/h96aayk/) on how that could be implemented, but since I don't use Wayland myself that would have to be added by contributors.

**Reply by john-ghatas** [reply]
No, Running on X11.

**Comment by mrothberg3** [comment]
Hi, I downloaded the AppImage for Linux for my Pop-OS 21.04 system and it installs perfectly.

When I run the app it displays: this in the terminal
INFO:root:Starting QtWebEngine
INFO:runekit.game.x11.manager.X11GameManager:Found game instance 48234516

But nothing pops up on my screen. Is there something I am missing that is preventing Alt1 from displaying?

**Reply by schoentoon** [reply]
Does it show up in your system tray? You'll have to launch the apps from there

**Reply by mrothberg3** [reply]
No, nothing shows up.

**Reply by mrothberg3** [reply]
The only thing I had to install before installing was QT which was accomplished via this command:

sudo apt-get install qtbase5-dev qtchooser qt5-qmake qtbase5-dev-tools

**Reply by mrothberg3** [reply]
I solved it by forcing it to open in my system tray using kdocker

**Comment by IG-Profileviewer** [comment]
I've installed it on Mac but when I open it, nothing happens. The app is open I see it in my dock but there's no overlay or anything

**Reply by schoentoon** [reply]
Does it show up in your system tray by any chance? It won't open any overlays by default, they have to be launched from the system tray.

**Comment by IG-Profileviewer** [comment]
I am an idiot. It was in my system tray. Thanks a lot!!!

**Comment by ghost** [comment]
Hey all, I posted a new question regarding Linux, https://github.com/whs/runekit/discussions/21

Cheers

**Comment by RyanColvin** [comment]
Everything working fine on Mac except on part of the clue solver. 

Saw a YouTube video where you can use triangulation on the compass clues. However when I right click and hover over a teleport name then click alt + 1 it doesn’t add it with the line on the mini map. 

I also just booted up Pop Os and wasn’t working on that either. 

Any suggestions?
Otherwise great app, thank you!

**Reply by schoentoon** [reply]
So what you could do instead (which is what I do with the compass steps on Ubuntu). Instead of doing alt +1, zoom in on the map and double click to plant down a marker. Afterwards press the button on top to draw the line. It isn't nearly as seamless as pressing alt + 1 and having it detect the teleport automatically etc, but at least it works.

**Comment by dd0014** [comment]
Hi everyone,

I’ve been using RuneKit on my Mac to assist with RuneScape, and it was working perfectly fine until recently. However, I’ve run into an issue where RuneKit is no longer able to read the RuneScape interface, and none of the apps like the Fish Flingers or Clue Solver are functioning correctly.

Here’s what I’ve tried so far:

	1.	Checked and Reset Permissions:
	•	I made sure that RuneKit has Full Disk Access and Screen Recording permissions in System Preferences > Security & Privacy. I also tried removing and re-adding RuneKit to these permissions, but it didn’t help.
	2.	Adjusted Display Settings:
	•	I tried running RuneScape in both fullscreen and windowed mode, lowered the resolution to 1080p, and disabled Retina scaling to see if it would improve screen capture detection, but no luck.
	3.	Cleared Cache and Configuration Files:
	•	I searched and cleared out any cache and configuration files related to RuneKit from ~/Library, but the problem persists even after a fresh start.
	4.	Reinstalled RuneKit:
	•	I uninstalled and reinstalled RuneKit, but the issue still remains.
	5.	Used the Default RuneScape Layout:
	•	I switched to the default RuneScape interface layout, which caused RuneKit to take a bit longer when trying to detect the interface, but it still couldn’t properly read the screen.
	6.	Checked Alt1 Capture Settings:
	•	I reviewed the capture problems guide from Alt1 and tried to apply similar principles, but since RuneKit on macOS doesn’t have explicit capture mode settings like Alt1, I’m unsure if the issue is related to OpenGL, DirectX, or something specific to macOS.

Has anyone else encountered this issue on macOS, and if so, how did you resolve it? Are there specific settings or workarounds that can help RuneKit correctly detect the RuneScape interface again? Any advice or insights would be greatly appreciated!

Thanks in advance!
